source/man/doveadm-user.1.rst.in

```diff
@@ -0,0 +1,148 @@
+.. _man-1_doveadm-user:
+
+============
+doveadm-user
+============
+
+NAME
+====
+
+doveadm-user - Perform a user lookup in Dovecot's userdbs
+
+SYNOPSIS
+========
+
+**doveadm** [**-Dv**] **user** [**-a** *userdb_socket_path*] [**-f** *field*] [**-u**] [**-x** *auth_info*] *user*...
+
+DESCRIPTION
+===========
+
+The **user** command is used to perform a user lookup - to show what
+information Dovecot sees about the *user* (s), or if it exists at all
+in the configured *userdb* (s).
+
+The *auth_info* may be useful when the userdb is for example a SQL
+database and you are using %variables, like **%s** or **%l**, in the
+*user_query* setting. Or when you have configured the userdb in a way
+like this:
+
+.. parsed-literal::
+
+   userdb {
+     driver = passwd-file
+     args = /etc/%s.passwd
+   }
+
+
+.. include:: global-options.inc
+
+
+Command specific *options*:
+
+**-a** *userdb_socket_path*
+   This option is used to specify an absolute path to an alternative
+   UNIX domain socket.
+
+By default :man:`doveadm(1)` will use the socket *@rundir@/auth-userdb*.
+The socket may be located in another directory, when the default
+*base_dir* setting was overridden in *@pkgsysconfdir@/dovecot.conf*.
+
+**-f** *field*
+   When this option and the name of a userdb field is given,
+   :man:`doveadm(1)` will show only the value of the specified field.
+
+**-u**
+   When this option is given, :man:`doveadm(1)` will only show values
+   from the *userdb*. Without -u parameter if any of the *uid*, *gid*,
+   *home* or *mail* fields are missing, their defaults are taken from
+   configuration file.
+
+**-x** *auth_info*
+   *auth_info* specifies additional conditions for the **user**
+   command. The *auth_info* option string has to be given as
+   *name* **=** *value* pair. For multiple conditions the **-x**
+   option could be supplied multiple times.
+
+   Possible names for the *auth_info* are:
+
+   **service**
+      The service for which the userdb lookup should be tested. The
+      value may be the name of a service, commonly used with Dovecot.
+      For example: **imap**, **pop3** or **smtp**.
+
+   **session**
+     Session identifier.
+
+   **lip**
+      The local IP address (server) for the test.
+
+   **rip**
+      The remote IP address (client) for the test.
+
+   **lport**
+      The local port, e.g. 143
+
+   **rport**
+      The remote port, e.g. 24567
+
+   **real_lip**
+      The local IP to which the client connected on this host.
+
+   **real_rip**
+      The remote IP where client connected from to this host.
+
+   **real_lport**
+      The local port to which client connected to to this host.
+
+    **real_rport**
+      The remote port from where the client connected from to this host.
+
+    **forward_<field>**
+      Field to forward as %{forward:field} to auth process.
```

Comment:
This is incorrectly indented (one space too many).

Reply:
Also `doveadm-auth` uses the `-x auth_info` option. This might be better off split off into its own `option-x-auth_info.inc` file, to allow a) consistent content, and b) reduced typing.